ast/functions_test.go

```diff
@@ -200,3 +203,15 @@ func (ts *testDMLSuite) TestWindowFuncExprRestore(c *C) {
 	}
 	RunNodeRestoreTest(c, testCases, "select %s from t", extractNodeFunc)
 }
+
+func (ts *testDMLSuite) TestNextValueForSequence(c *C) {
+	testCases := []NodeRestoreTestCase{
+		{"next value for seq", "NEXTVAL(`seq`)"},
+		{"next value for sequence", "NEXTVAL(`sequence`)"},
+		{"NeXt vAluE for seQuEncE2", "NEXTVAL(`seQuEncE2`)"},
+	}
+	extractNodeFunc := func(node Node) Node {
+		return node.(*SelectStmt).Fields.Fields[0].Expr
+	}
+	RunNodeRestoreTestWithoutCompareAST(c, testCases, "select %s", extractNodeFunc)
```

Comment:
Both `NEXT VALUE FOR seq ` and `nextval(seq)` should produce the same AST. This means `nextval` and friends should be special-cased to only accept ObjectNameExpr (the `seq` in `nextval(seq)` should never mean a column named `seq`)

Reply:
Exactly right~ I see, got it!

Reply:
Exactly right, got it~